FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,46 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: self.url) { [weak self] (result) in
+			guard self != nil else { return }
+			switch result {
+			case .failure(_):
+				completion(.failure(Error.connectivity))
+			case .success(let (data, httpResponse)):
+				do {
+					let feed = try FeedImageMapper.map(data, httpResponse)
+					completion(.success(feed))
+				} catch {
+					completion(.failure(Error.invalidData))
+				}
+			}
+		}
+	}
+}
+
+private class FeedImageMapper {
+	static private var OK_200 = 200
```

Comment:
Access level should precede the `static` keyword.

```suggestion
	private static var OK_200 = 200
```